crates/projection/tests/projection.rs

```diff
@@ -0,0 +1,304 @@
+use approx::assert_relative_eq;
+use nalgebra::{point, vector, Isometry3, Point2, Translation, UnitQuaternion, Vector2};
+use projection::Projection;
+use types::CameraMatrix;
+
+fn from_normalized_focal_and_center_short(
+    focal_length: Vector2<f32>,
+    optical_center: Point2<f32>,
+    image_size: Vector2<f32>,
+) -> CameraMatrix {
+    CameraMatrix::from_normalized_focal_and_center(
+        focal_length,
+        optical_center,
+        image_size,
+        Isometry3::identity(),
+        Isometry3::identity(),
+        Isometry3::identity(),
+    )
+}
+
+#[test]
+fn pixel_to_camera_default_center() {
```

Comment:
Did you copy all tests? I would like to not check them manually if they only have been copied.